packages/create-audius-app/tests/e2e/react.test.ts

```diff
@@ -0,0 +1,66 @@
+import { test, expect } from '@playwright/test'
+
+test('auths, fetches tracks, and favorites a track', async ({
+  page,
+  context
+}) => {
+  test.slow()
```

Comment:
was dis?

Reply:
Because it waits for the block confirmation, we need to extend the timeout. `test.slow()` triples the default timeout which is 5000ms in our case

Reply:
Thats super neat, I didn't know about this! 
I've just been changing this in the config. If you wanted to use a more custom number changing it in the config is also option